src/config/wagmi-config.ts

```diff
@@ -0,0 +1,16 @@
+import { configureChains } from '@wagmi/core';
+import { linea, lineaTestnet } from '@wagmi/core/chains';
+import { createConfig } from 'wagmi';
+import { MetaMaskConnector } from 'wagmi/dist/connectors/metaMask';
+import { publicProvider } from 'wagmi/dist/providers/public';
+
+const { chains, publicClient } = configureChains(
+  [linea, lineaTestnet],
+  [publicProvider()],
+);
+
+export const wagmiConfig = createConfig({
```

Comment:
We use upper case snake case for constants.
```suggestion
export const WAGMI_CONFIG = createConfig({
```